### ISSUE TITLE
'make check' failure

### ISSUE THREAD

**Issue by dmacks** [lsh123/xmlsec#465]
Building xmlsec-1.2.37 on OS X 10.13, with gcrypt/gnutls/gpg-error/openssl3 supplied via fink:

```
$ ./configure --prefix=/sw --enable-aes --disable-gost --without-nss --disable-apps-crypto-dl --disable-static --enable-dependency-tracking --with-html-dir=/sw/share/doc/xmlsec1-dev --libdir=/sw/lib/xmlsec1
[all the expected things found]
$ make
[no errors or warnings]
$ export TMPFOLDER=/sw/build.build/xmlsec1-shlibs-1.2.37-2/xmlsec1-1.2.37/tests/tmp
$ mkdir $TMPFOLDER
$ make check
for crypto in openssl gcrypt gnutls  ; do \
                make check-crypto-$crypto || exit 1 ; \
        done
[all 'OK' or 'Skip' until...]
--- testKeys finished ---
--- detailed log is written to  /sw/build.build/xmlsec1-shlibs-1.2.37-2/xmlsec1-1.2.37/tests/tmp/testKeys.sh.20230129_200652-4979.log ---
--- TOTAL OK: 0; TOTAL FAILED: 0; TOTAL SKIPPED: 7
--- testKeys started for xmlsec-gcrypt library (20230129_200652) ---
--- LD_LIBRARY_PATH=
--- LTDL_LIBRARY_PATH=
Test: hmac-192 (success)
 /sw/build.build/xmlsec1-shlibs-1.2.37-2/xmlsec1-1.2.37/apps/xmlsec1 check-key-data  --crypto gcrypt --crypto-config /sw/build.build/xmlsec1-shlibs-1.2.37-2/xmlsec1-1.2.37/tests/tmp/xmlsec-crypto-config hmac
Error: dynamic xmlsec-crypto library loading is disabled and the only available crypto library is 'openssl'
Unknown command
Usage: xmlsec <command> [<options>] [<files>]
[xmlsec usage-instructions]
[repeated several more times]
--- testKeys finished ---
--- TOTAL OK: 0; TOTAL FAILED: 0; TOTAL SKIPPED: 7
make[1]: *** [check-crypto-gcrypt] Error 1
make: *** [check-all] Error 1
```

One problem appears to be it's running xmlsec using an invalid set of parameters (or at least inconsistently with the support libraries it detected). And a second situation is that having every test skipped (and no failures) is not considered a successful outcome. That second sounds logically flawed. But it's apparently intentional, based on [this commit to testrun.sh](https://github.com/lsh123/xmlsec/pull/373/files#diff-ccc562ff1088d8622674d91abc2c3ed107920dc9ac4350c143939949983259db)  around line 500

**Comment by lsh123** [comment]
Thanks, this should be fixed shortly in https://github.com/lsh123/xmlsec/pull/467

**Comment by lsh123** [comment]
Ops, closed this issue by mistake. The second problem (tests don't fail if nothing succeeded) is addressed in PR #496
